pkg/inframanager/p4/service_es2k.go

```diff
@@ -249,9 +248,9 @@ func InsertServiceRules(ctx context.Context, p4RtC *client.Client,
 		actn = Insert
 		groupID = uuidFactory.getUUID()
 		service.GroupID = groupID
-		epNum = 0
 	}
 
+	epNum = service.NumEndPoints
```

Comment:
The error was of possible mod by 0 on line 402. If the for statement on 260 didn't execute, then epNum would have remained 0.

Reply:
I think this is the wrong thing to do since the epNum is supposed to overwrite the previous value if it is an add and not an update. Does it make sense to return an error if ` len(podIpAddr) == 0` from this func? Is there any condition where no Pod IPs would be given? mAYBE DIFFERENTLY FOR UPDATE AND ADD SCENARIOS?